### PR TITLE
Add weather and clock

### LINKED ISSUE #7: Add script to get time at a location
I have no clue how to do this but it's a good script idea, hence opening an issue. 

We can have a script that takes in place info (example - `PST`, `Chicago`, `London`) and gives the current time in that location. One way to do it could be scraping Google or using a REST API but there could be native language libraries that do this too.

#5

### PR DESCRIPTION
**Pull request by missingcharacter**
Improvements to https://github.com/aviaryan/utility-bash-scripts/pull/11 to fix https://github.com/aviaryan/utility-bash-scripts/issues/7

------

**I have checked the following requirements before making this PR.**

- [x] The script is marked as executable (`chmod +x`).
- [x] I have added/updated the README entry for the script (if applicable).
- [x] I have mentioned any dependencies and OS requirements for the script in the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -146,3 +146,22 @@ Script: [rm-ds-store](rm-ds-store)
...
+
+```sh
+clock list
+clock <Time Zone>
+```
+
+### 🌧 Get weather forecast
+
+Script: [weather](weather)
+Dependencies: [curl](https://curl.haxx.se/)
```
Comment (aviaryan) [inline README.md]: I think `curl` is pre-installed on most of the system. Am I wrong?
If no, maybe we can remove it to keep the README clean.

Comment (aviaryan) [reply README.md]: Maybe I should update the PR template to `...mentioned any outstanding dependencies...`.

**Comment by aviaryan** [comment]
Nice work again @missingcharacter. Really appreciate it. 👏

**Comment by missingcharacter** [comment]
:)

**Pull request merged**